### PR TITLE
Add r-dicekriging

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

**Comment by jschueller** [comment]
ping @conda-forge/core

File: recipes/r-dicekriging/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+test:
+  commands:
+    - $R -e "library('DiceKriging')"  # [not win]
+    - "\"%R%\" -e \"library('DiceKriging')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=DiceKriging
+  license: GPL-2 | GPL-3
+  license_family: GPL
```
Comment (scopatz) [inline recipes/r-dicekriging/meta.yaml]: For GPL, you need to provide the licence file

**Comment by scopatz** [comment]
Thanks for putting this in @jschueller!

**Comment by jschueller** [comment]
ping @conda-forge/core

File: recipes/r-dicekriging/meta.yaml (hunk 1 above)
Comment (jschueller) [reply recipes/r-dicekriging/meta.yaml]: ok, added

**Pull request merged**